BuildResidentialHPXML/resources/schedules.rb

```diff
@@ -87,6 +87,18 @@ def self.col_names
     ]
   end
 
+  def self.unaffected_by_vacancy
+    return [
+      'refrigerator',
+      'extra_refrigerator',
+      'freezer',
+      'pool_pump',
+      'pool_heater',
+      'hot_tub_pump',
+      'hot_tub_heater'
+    ]
```

Comment:
Seems better to return a hash where every end use is accounted for, with true or false as appropriate. And then give an error if there's an end use not covered by the hash. That way this code would be updated if an end use name changes or a new end use is added.

Reply:
Good idea. I will update.